grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1952,7 +1952,83 @@ def section_frontend(outer_panels):
                     ],
                     ["last"],
                 ),
-                panels.timeseries_latency(
+                panels.timeseries_count(
+                    "Subsription Cursor Nums",
+                    "The number of valid and invalid subscription cursor",
+                    [
+                        panels.target(
+                            f"{metric('valid_subsription_cursor_nums')}",
+                            "",
+                        ),
+                        panels.target(
+                            f"{metric('invalid_subsription_cursor_nums')}",
+                            "",
+                        ),
+                    ],
+                ),
+                panels.timeseries_count(
+                    "Subscription Cursor Error Count",
+                    "The subscription error num of cursor",
+                    [
+                        panels.target(
+                            f"{metric('subscription_cursor_error_count')}",
+                            "",
+                        ),
+                    ],
+                ),
+                panels.timeseries_count(
+                    "Subscription Cursor Query Duration(ms)",
+                    "The amount of time a query exists inside the cursor",
+                    [
+                        *quantile(
+                            lambda quantile, legend: panels.target(
+                                f"histogram_quantile({quantile}, sum(rate({metric('subscription_cursor_query_duration_bucket')}[$__rate_interval])) by (le, cursor_name))",
+                                f"p{legend} - {{{{cursor_name}}}}",
```

Comment:
Having `cursor_name` as the metric label will create too many labels. I suggest we either have an aggregated metric for all cursors or use `subscription_name` instead.